storage/innobase/ibuf/ibuf0ibuf.cc

```diff
@@ -38,6 +38,7 @@ this program; if not, write to the Free Software Foundation, Inc.,
 #include "sync0sync.h"
 
 #include "my_dbug.h"
+#include "sql/raii/sentry.h"
```

Comment:
Perhaps it is better to use include/scope_guard.h instead to avoid introducing one more dependency to InnoDB on something from sql/ directory?

Reply:
I was looking for this and searching with all kinds of names. Glad you found it. Thanks!